### DISCUSSION TITLE
A developmental trap?

### DISCUSSION THREAD

**Discussion by Nick222** [widelands/wl_addons_server#114] [Q&A] [answered]
Hello!

Unfortunately, I just can't figure out how to get miners of normal level and above.

By the time basic level miners finish mining (since the mine is depleted), their experience is still too low to level up.

At the same time, the "personnel department" also does not have time to develop to the required level and remains basic.

How should the player behave correctly in order not to get into this "dead end"?

Thanks :)

**Comment by klaus-halfmann** [comment, marked as answer]
Hello @Nick222 welcome to widelands.

On any normal Map Miners should be able to gain enough experience before you need to upgrade the mine.
On a few maps resources are low, so you must keep them in the depleted mine until (which takes much longer)
they have found some "random" resources and level up.

Please attach a savegame so we may see where this happend.
Please also include the verision of widelands etc.

Please be sure that you played _all_ the tutorials.

Pleas check also with the main widelands [forums](https://www.widelands.org/forum/)
and eventually some videos on youtube, e.g. https://www.youtube.com/watch?v=VmHs3D9vYEc&list=PLDOZxhyIjga6LYmvfBJTZjwt_vKRZ5L0n

**Comment by Noordfrees** [comment]
In most mines, the miners gain experience even when the mine is depleted and they find nothing, though at a much slower rate than usually. So you can just keep the mine running and watch the miners slowly level up.

The exact values are different for each tribe. Were you playing Europeans or an official tribe? If Europeans, this might be a balancing problem specific to them, the experience thresholds are very high given the low yield percentage of the lower-level mines.
